resources/js/googleMaps.js

```diff
@@ -0,0 +1,66 @@
+function initAutocomplete() {
+    new google.maps.places.Autocomplete(
+        document.getElementById('locationName'));
+}
+
+function initMap() {
+    const map = new google.maps.Map(document.getElementById("map"), {
+        zoom: 8,
+        center: {lat: 44.4267674, lng: 26.1025384}
+    });
+    var locationAddress = document.getElementById('locationForm');
+    locationAddress.addEventListener('submit', geocode);
+    function geocode(e) {
+        //Previne trimiterea form-ului
+        e.preventDefault();
+        var location = document.getElementById('locationName').value;
+        axios.get("https://maps.googleapis.com/maps/api/geocode/json", {
+            params: {
+                address: location,
+                key: 'AIzaSyBB0Tj2YoWLm7ozC9I9P_zkESQeRmCKDXw'
+            }
+        }).then(function (response) {
+            //Afisam tot raspunsul primit in consola
+            console.log(response);
+            //Locatia
+            var lat = response.data.results[0].geometry.location.lat;
+            var lng = response.data.results[0].geometry.location.lng;
+            var afisareaLatLng = `
+            <ul>
+                <li>${lat}</li>
+                <li>${lng}</li>
+            </ul>
+            `;
+            //Adresa formatata
+            var adresaFormatata = response.data.results[0].formatted_address;
+            var afisareadresaFormatata = `
+            <ul>
+                <li> ${adresaFormatata}</li>
+            </ul>
+            `;
+            //Afisam pe ecran
+            document.getElementById('adresaFormatata').innerHTML = afisareadresaFormatata;
+            document.getElementById('position').innerHTML = afisareaLatLng;
+            //Centram harta acolo unde a bagat utilizatorul
+            map.setCenter({
+                lat: response.data.results[0].geometry.location.lat,
+                lng: response.data.results[0].geometry.location.lng
+            });
+            // The marker
+            const marker = new google.maps.Marker({
+                position: {
+                    lat: response.data.results[0].geometry.location.lat,
+                    lng: response.data.results[0].geometry.location.lng
+                },
+                map: map,
+            });
+        }).catch(function (error) {
+            console.log(error);
+        });
+    }
+    initAutocomplete();
+}
+
+window.onload = function() {
+    initMap();
+};
```

Comment:
same here, as above

Reply:
I don't understand what is wrong here

Reply:
delete romanian comments and check indentation